client/my-sites/stats/controller.jsx

```diff
@@ -268,8 +265,9 @@ module.exports = {
 		} else if ( ! activeFilter || -1 === validModules.indexOf( context.params.module ) ) {
 			next();
 		} else {
-			if ( 'object' === typeof( site.options ) && 'undefined' !== typeof( site.options.gmt_offset ) ) {
-				momentSiteZone = i18n.moment().utcOffset( site.options.gmt_offset );
+			const gmtOffset = getSiteOption( context.store.getState(), siteId, 'gmt_offset' );
+			if ( Number.isFinite( gmtOffset ) ) {
```

Comment:
Is there a reason above we use a truthy ternary `gmtOffset ? gmtOffset : 0` and here `Number.isFinite`? Should we be consistent?